.cirrus.yml

```diff
@@ -964,7 +964,7 @@ image_build_task: &image-build
             CTXDIR: contrib/podmanimage/stable
         - env:
             CTXDIR: contrib/hello
-    env:
+    env: &image-build-cred-envs
```

Comment:
@cevich i'm hoping to reuse the envs in the fcos-image-build task, but CI says the `PODMAN_` variables are referenced but not assigned. I'd have imagined the `&image-build` on top of the task should've sufficed, but apparently not (?).

Reply:
Knowing what we know now, those vars are useless.  Since the actual build+push will have to happen in another automation system, I think you can keep this task super-simple.  All it needs to do is a test-build of the image.